### ISSUE TITLE
Substition alias no longer works since 2071

### ISSUE THREAD

**Issue by WhiskyTangoFawks** [MinecraftForge/MinecraftForge#3219]
It looks like the fix for issue #3212 (prev issue I raised about aliases and servers) broke the alias system.

Just updating forge, not modifying my code, the replaced block now has a null model, and the replaced class's methods are not called (evidenced by the block returning vanilla light levels and having particule animations, which my class's methods don't).  No errors are raised, and a call to the Blocks.torch.getClass (which is the block I'm replacing) returns my replacement class.  The block displays correctly in the inventory, just not in the world.

In 2070, the system worked fine, aside from the server issue, which I raised in #3212.

Thanks

**Comment by LexManos** [comment]
Write me a full test mod.
This is getting ridiculous.
All I did was make it so that the map in question gets reset. I verified the map and registry state get setup correctly.
Resource packs are not reloaded when connecting to the server so everything _should_ be fine.

**Comment by WhiskyTangoFawks** [comment]
Will this work? 
https://github.com/WhiskyTangoFawks/TestMod/tree/master/testmod/src/main/java/testmod/aliastest

It just does torch aliasing, with one with a lightvalue of 0, and that shouldn't have flame particles .

I've also included a method on player right click which just prints out to chat the name of the block and item when you right click.

While doing this, I noticed that if you place the aliased block in game, you get a null model, and the incorrect block (so, in this example, you get a vanilla torch with a null model), but if you then exit the game and reload, it updated to the aliased block, with a correct model.  The item in hand for the block also returns the vanilla torch (I tried doing replacement of both block and item, but that causes an existing alias error- I'm guessing it's been automated to do both when you put a block in?).

Let me know if there's anything else I can do to help.
